portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```
@@ -29,25 +30,70 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.text.ParseException;
+import java.util.TimeZone;
+import java.util.Calendar;
+import java.text.DateFormat;
 
 @WebServlet("/comment-section")
 public class DataServlet extends HttpServlet {
 
+  Integer defaultCommentsNumber = 4;
+  SimpleDateFormat DateFor = new SimpleDateFormat("dd/MM/yy kk:mm:ss");
+  TimeZone timeZone = TimeZone.getTimeZone("Europe/Bucharest");
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment");
+    String commentsOrder;
+    try {
+      commentsOrder = request.getParameter("commentsorder");
+      if (commentsOrder.equals("DESC")) {
+        query.addSort("timestamp", SortDirection.DESCENDING);
+      } else {
+        query.addSort("timestamp", SortDirection.ASCENDING);
+      }
+    } catch (Exception e) {
```

See comment below regarding not ignoring exceptions, but returning an error back to the caller.